package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@endpass/connect-demo",
-  "version": "1.2.1",
+  "version": "1.3.0",
```

Comment:
May be `2.0.0`? Redesign is not minor patch

Reply:
Functional was not changed